parse_test.go

```diff
@@ -78,6 +78,7 @@ func checkDuplicationWithinResults(t *testing.T, result shared.MetadataTestResul
 	}
 	expected := serializeStrings(result.TestPath, subtestName, status)
 	assert.False(t, resultSet.Contains(expected), "duplicated entries within results")
+	resultSet.Add(result.TestPath)
```

Comment:
I don't think this works - you're only checking for duplication in one direction. Consider the following:

```
links:
- product: chrome
  url: https://whatever.com
  results:
  - test: font-synthesis-01.html
  - test: font-synthesis-01.html
    status: FAIL
```

If you want to catch these cases, you need to consider no-subtest and/or no-status* to be wildcards, and change the test logic to handle that.

* I think it makes some sense to make no-status a wildcard, I'm less convinced about subtests. Is it really invalid to triage both a test in general and some of its subtests?

Reply:
Good catch and I agree that this makes less sense when subtests are involved. On a second thought, also it might be good keep the current structure because a test could have multiple statues due to different browser versions, different WPT test versions or test flakiness. I will figure out a stricter GitHub checks when the flaky metadata is introduced